src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -40,6 +40,8 @@ class OTARequestor : public OTARequestorInterface, public BDXDownloader::StateDe
     OTARequestor() : mOnConnectedCallback(OnConnected, this), mOnConnectionFailureCallback(OnConnectionFailure, this) {}
 
     //////////// OTARequestorInterface Implementation ///////////////
+    void Shutdown(void) override;
```

Comment:
This is very much platform-specific, so I have a feeling it shouldn't be part of OTA requestor interface, but I'm not sure how to trigger server shutdown in a better way...

Reply:
Yeah, I was having a hard time figuring out where best to put this. Perhaps a follow up for post v1 to determine a better way to shutdown the server?

Reply:
Yes, I think this should be cleaned up at some point since embedded platforms typically don't need to implement any graceful shutdown involving killing threads etc.